tox.ini

```diff
@@ -143,24 +127,7 @@ commands_pre =
     pip install -r {env_tmp_dir}/requirements.txt
     pip freeze
 commands =
-    pytest asdf-astropy \
-        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.tests.helpers"
-
-[testenv:specutils]
-change_dir = {env_tmp_dir}
-allowlist_externals =
-    git
-    bash
-extras =
-commands_pre =
-    bash -c "pip freeze -q | grep 'asdf @' > {env_tmp_dir}/requirements.txt"
-    git clone https://github.com/astropy/specutils.git
-    pip install -e specutils[test]
-    pip install -r {env_tmp_dir}/requirements.txt
-    pip freeze
-commands =
-    pytest specutils \
-        -W "ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.tests.helpers"
```

Comment:
Leave this in because we may want to add specutils back once its extension has been modernized. This will let us easily test that.

Reply:
Sounds good.